### ISSUE TITLE
NavBrand: insufficient color contrast

### ISSUE THREAD

**Issue by danethurber** [pluralsight/design-system#1580]
per our audit - related to the NavBrand component

![Screen Shot 2021-02-01 at 10 50 23 AM](https://user-images.githubusercontent.com/226356/106497490-57b27400-647b-11eb-91d9-4f0533499ce7.png)

> The "Skills" button has an insufficient color contrast of 1.4:1 in hover state (foreground color: #292F34, background color: #0D0F12). Expected color ratio is 3:1.
> Please follow next link to find the correct colors for hover state to reach the expected ratio: 
https://contrast-finder.org/

**Comment by sweeting** [comment]
@danethurber 

From: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

> **Boundaries**
> This success criteria does not require that controls have a visual boundary indicating the hit area, but if the visual indicator of the control is the only way to identify the control, then that indicator must have sufficient contrast. If text (or an icon) within a button or placeholder text inside a text input is visible and there is no visual indication of the hit area then the Success Criterion is passed. 

If I am understanding this correctly, the hover state background of the NavBrand item does not need to meet 3:1 contrast ratio because the image inside NavBrand does meet contrast guidelines. This would also apply to Secondary & Flat buttons.
